services/accounts/accounts.go

```diff
@@ -429,7 +441,7 @@ func (api *API) VerifyPassword(password string) bool {
 	return err == nil
 }
 
-func (api *API) AddMigratedKeyPair(ctx context.Context, kcUID string, kpName string, keyUID string, accountAddresses []string, keyStoreDir string) error {
+func (api *API) AddMigratedKeyPair(ctx context.Context, kcUID string, kpName string, keyUID string, accountAddresses []string, password string) error {
```

Comment:
This method is not used on mobile so no problem here.

Reply:
Great! ;)